src/styles/clock.scss

```diff
@@ -0,0 +1,59 @@
+.clock {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
instead of applying transform, try this
```
top: calc(50% - 300px)
```
300px - half of the height

the same with left